tests/Get-DbaTraceFlag.Tests.ps1

```diff
@@ -3,52 +3,37 @@ Write-Host -Object "Running $PSCommandpath" -ForegroundColor Cyan
 . "$PSScriptRoot\constants.ps1"
 
 Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
-	Context "Verifying output" {
+	Context "Verifying TraceFlag output" {
 		BeforeAll {
-			$tfs = 3226,3604
-			Write-Host "Quick check on current global TFs enabled"
-			$server = Connect-DbaSqlServer -SqlInstance $script:instance1
-			$checkResults = $server.Query("DBCC TRACESTATUS(-1)")
-			if ($checkResults.Count -gt 0) {
-				Write-Host "Found $($checkResults.Count) global TFs enabled, need to disable them" -ForegroundColor Cyan
-				foreach ($result in $checkResults) {
-					$tf = $result.TraceFlag
-					$sql = "DBCC TRACEOFF($tf,-1);"
-					$server.Query($sql)
-				}
+			$safetraceflag = 3226
```

Comment:
3604 is a safe TF to use, it is just used to print DBCC output to the query window. Which in this case will never be utilized because it is all CL.